src/back/msl/mod.rs

```diff
@@ -282,6 +293,32 @@ impl Options {
             None => Err(EntryPointError::MissingPushConstants(stage)),
         }
     }
+
+    fn resolve_sizes_buffer(
+        &self,
+        stage: crate::ShaderStage,
+    ) -> Result<ResolvedBinding, EntryPointError> {
+        let slot = match stage {
+            crate::ShaderStage::Vertex => self.per_stage_map.vs.sizes_buffer,
```

Comment:
if you feel fancy, you could implement `Index<crate::ShaderStage> for PerStageMap`

Reply:
That's a good idea, I'll do that.